qdev_wrappers/customised_instruments/SR830_ext.py

```diff
@@ -37,81 +34,73 @@ def get(self):
 # Subclass the SR830
 
 class SR830_ext(SR830):
-    """
-    An SR830 with the following super powers:
-        - a Voltage divider
-        - An I/V converter
-        - A conductance buffer
-    """
-
-    def __init__(self, name, address, config, **kwargs):
+    def __init__(self, name, address, **kwargs):
         super().__init__(name, address, **kwargs)
 
-        # using the vocabulary of the config file
-        self.ivgain = float(config.get('Gain Settings',
-                                       'iv gain'))
-        self.__acf = float(config.get('Gain Settings',
-                                      'ac factor'))
-
-        self.add_parameter('amplitude_true',
-                           label='ac bias',
-                           parameter_class=VoltageDivider,
-                           v1=self.amplitude,
-                           division_value=self.acfactor)
-
-        self.acbias = self.amplitude_true
+        self.add_parameter(name='iv_gain',
+                           label='I/V Gain',
+                           unit='',
+                           set_cmd=lambda x: x,
+                           get_parser=float)
 
-        self.add_parameter('g',
-                           label='{} conductance'.format(self.name),
-                           # use lambda for late binding
+       	self.add_parameter(name='g',
+                           label='Conductance',
+                           unit='e$^2$/h',
                            get_cmd=self._get_conductance,
-                           unit='e^2/h',
                            get_parser=float)
 
-        self.add_parameter('conductance',
-                           label='{} conductance'.format(self.name),
-                           parameter_class=ConductanceBuffer)
-
-        self.add_parameter('resistance',
-                           label='{} Resistance'.format(self.name),
+       	self.add_parameter(name='resistance',
+                           label='Resistance',
+                           unit='Ohm',
                            get_cmd=self._get_resistance,
+                           get_parser=float)
+        
+        self.add_parameter(name='g_X',
```

Comment:
This indentation seems to be off

Reply:
Or rather above, I think some taps have ended up in the code

Reply:
Weird. It still runs though. Tried redoing the indentation but feel like there is an automatic way of doing that which I haven't learned yet...

Reply:
Depending on the editor there is usually a convert taps to space setting and a indentation config